.github/workflows/build.yml

```diff
@@ -53,6 +53,13 @@ jobs:
         run: |
           cmake --build build --target install
           cmake --build build --target appimage
+      - name: Display help
+        run: >
+          build/install/bin/lmms --help | grep "Usage: lmms"
```

Comment:
```suggestion
        run: build/install/bin/lmms --help | grep "Usage: lmms"
```

Reply:
Hm weird, the yamllint CI says:
```
  Error: 57:57 syntax error: mapping values are not allowed here (syntax)
```
Possibly because of the `|`?